src/components/modalForEdit/ModalForEdit.tsx

```diff
@@ -0,0 +1,265 @@
+            <input
+              type='text'
+              className='input_for_title'
+              placeholder='Title'
+              value={bookForUpdate.Title}
+              onChange={({ target }) => setBookForUpdate((prevTitle) => ({ ...prevTitle, Title: target.value }))}
+            />
+          </div>
+          <div className='container_for_element'>
+            <input
+              type='text'
+              className='input_for_description'
+              placeholder='Description'
+              value={bookForUpdate.Description}
+              onChange={({ target }) => setBookForUpdate((prevDescription) => ({ ...prevDescription, Description: target.value }))}
+            />
+          </div>
+          <div className='container_for_element'>
+            <input
+              type='text'
+              className='input_for_isbn'
+              placeholder='ISBN'
+              pattern='^(?=(?:\D*\d){10}(?:(?:\D*\d){3})?$)[\d-]+$'
+              value={bookForUpdate.ISBN}
+              onChange={({ target }) => setBookForUpdate((prevIsbn) => ({ ...prevIsbn, ISBN: target.value }))}
+            />
+          </div>
+          <div className='container_for_element'>
+            <input
+              type='text'
+              className='input_for_quantity'
+              placeholder='Quantity'
+              value={bookForUpdate.Quantity}
+              onChange={({ target }) => setBookForUpdate((prevQuantity) => ({ ...prevQuantity, Quantity: +target.value }))}
+            />
+          </div>
+          <div className='container_for_element'>
+            <input
+              type='date'
+              className='input_for_date'
+              value={bookForUpdate.PublishDate ? new Intl.DateTimeFormat('en-CA').format(new Date(bookForUpdate.PublishDate)) : ''}
+              onChange={({ target }) => setBookForUpdate((prevDate) => ({ ...prevDate, PublishDate: target.value }))}
+            />
+          </div>
+          <div className='container_for_element'>
+            <input type='file' className='input_for_image' onChange={handleImgUpload} />
+          </div>
+          <div className='container_for_element'>
+            <Select
+              name='authors'
+              id='authors'
+              className='select_for_authors'
+              getOptionLabel={(author : AuthorResponse) => `${author.FirstName} ${author.LastName}`}
+              isMulti
+              getOptionValue={(author : AuthorResponse) => author.Id.toString()}
+              options={authorList}
+              onChange={handleSelectedAuthorsChange}
+              value={selectedAuthors}
+            />
+            <div className='add_book_button' >
+              <button
+                type='button'
+                onClick={() => setToggleAuthorForm(false)}
+                className={ toggleAuthorForm ? 'button_for_add_book' : 'hiden_add_button'}
+              >
+                Add book <BookAddIcon/>
```

Comment:
Clicking on add book and book icon opens inputs for adding author...

Reply:
I fixed this.